sdk/synapse/Microsoft.Azure.Management.Synapse/src/Generated/SdkInfo_SynapseManagementClient.cs

```diff
@@ -13,50 +13,15 @@ namespace Microsoft.Azure.Management.Synapse
 
```

Comment:
Did you change the name of your RP?
I think you need to get rid of this file to unblock build

Reply:
We added a new client named `SynapseSqlV3ManagementClient`. I made some change to fix the build issue.

Reply:
OK. So please add test cases for the new client. Thanks